poll/public/handlebars/poll_results.handlebars

```diff
@@ -1,13 +1,13 @@
 <script id="poll-results-template" type="text/html">
     <h3 class="poll-header">{{display_name}}</h3>
     <div class="poll-question-container">{{{question}}}</div>
-    <ul class="poll-answers-results poll-results">
+    <ul class="poll-answers-results poll-results {{~#if any_img}} has-images{{/if}}">
     {{#each tally}}
         <li class="poll-result">
             <div class="poll-result-input-container">
               <input id="answer-{{key}}" type="radio" disabled {{#if choice}}checked{{/if}} />
             </div>
-            {{~#if any_img~}}
+            {{~#if ../any_img~}}
```

Comment:
@Kelketek Apparently, underscore templates have this weird "feature" where template code within a loop context (`#each tally` in this case) does not "see" the outer variable context by default. So `any_img` is not defined at all within the loop, unless you explicitly add the `../` to indicate that you want to read a variable from the parent scope.

Took me a while to figure that out, as it made no sense to me at first.